src/deepsparse/server/tune.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+
+@click.command(context_settings=dict(show_default=True))
+@click.argument("config_path", type=click.Path(exists=True, dir_okay=False))
+@click.option(
+    "-e",
+    "--endpoint",
+    required=True,
+    type=str,
+    help="The name of the endpoint to tune.",
+)
+@click.option(
+    "-c",
+    "--concurrency",
+    required=True,
+    type=int,
+    help="The expected amount of concurrent requests.",
+)
+@click.option(
+    "-b",
+    "--batch-size",
```

Comment:
Should this and `concurrency` be ranges instead of single values?

And related to this, what use cases do people have where they will be sending batches of input to the server? It is easy to imagine just sending a single unbatched item to the server.

Reply:
(Take this comment with a grain of salt) but sometimes runtime engines give a greater performance improvement with a bigger `batch_size`; But more than that I think of it this way:

Supporting a batch_size > 1 will allow users to simply swap out the forward pass in their existing codebases with a call to the deepsparse server w/o changing any data-loading logic or unwrapping a batch?
(Thus making it easier to try deepsparse?)

Does that make sense?